### PR TITLE
Ethers core

### PR DESCRIPTION
**Pull request by KBryan**

### CODE REVIEW

**Review by sneakzttv — APPROVED** [review]
core dll woo

File: Scenes/SampleScene.unity [hunk 1]
```diff
@@ -0,0 +1,303 @@
+%YAML 1.1
```
Comment (FSM1) [inline Scenes/SampleScene.unity:1]: Any reason this is being added to the repo?

File: Resources/BillingMode.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```
Comment (FSM1) [inline Resources/BillingMode.json:1]: Same here

Comment (FSM1) [inline Resources/BillingMode.json:1]: Same here

Comment (FSM1) [inline Resources/BillingMode.json:1]: Any reason this is being added to the repo?

**Review by FSM1 — CHANGES_REQUESTED** [review]
A couple of files made it in to the repo that I think don't necessarily have to be there.

**Review by sneakzttv — APPROVED** [review]

**Pull request merged**